.github/workflows/pull_request.yml

```diff
@@ -10,6 +10,8 @@ env:
 jobs:
   quality_checks:
     uses: ./.github/workflows/quality_checks.yml
+    secrets:
+      SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
```

Comment:
how did you get this token? How long is it valid for? How do we renew it?
can we add instructions to readme/confluence on this